test/import-map.test.js

```diff
@@ -0,0 +1,106 @@
+const request = require("supertest");
+const { app, setConfig } = require("../src/web-server");
+
+beforeAll(() => {
+  setConfig({
+    manifestFormat: "importmap",
+    username: "username",
+    password: "password",
+    locations: {
+      prod: "memory://prod",
+    },
+  });
+});
+
+describe(`/import-map.json`, () => {
+  it(`does not return anything when it's not setup yet.`, async () => {
+    const healthResponse = await request(app)
+      .get("/import-map.json")
+      .expect(200)
+      .expect("Content-Type", /json/);
+
+    // we did not setup yet
+    expect(healthResponse.body.message).toBe(undefined);
```

Comment:
Shouldn't this return an empty import map? Since if the io-method returns an empty string, we create an empty import map.

Reply:
Nope its something i’ve noticed twice setting up the project, would be nice to solve 😊

Reply:
Would you, @joeldenning, want to fix that in this pr, or create a new pr for that one?